src/SUMMARY.md

```diff
@@ -76,7 +76,8 @@
 - [Welcome](welcome-day-2.md)
 - [Pattern Matching](pattern-matching.md)
   - [Matching Values](pattern-matching/match.md)
-  - [Destructuring](pattern-matching/destructuring.md)
+  - [Structs](pattern-matching/Structs.md)
+  - [Enums](pattern-matching/Enums.md)
```

Comment:
Please use lower-case letters for the filenames:

```suggestion
  - [Structs](pattern-matching/structs.md)
  - [Enums](pattern-matching/enums.md)
```

Reply:
Even better, please add "Destructuing" to the title here and below:

```suggestion
  - [Structs](pattern-matching/destructuring-structs.md)
  - [Enums](pattern-matching/destructuring-enums.md)
```

Please also make sure to add redirections in `book.toml` to preserve the existing URLs. This is important for search engine optimization (SEO) so people can continue to easily find the information.

Reply:
Need help with adding redirections in `book.toml` .